HW05.Rmd

```diff
@@ -0,0 +1,153 @@
+  *Each variable must have its own column.
+  *Each observation must have its own row.
+  *Each value must have its own cell.
+  
+```{r dad_mom tidying}
+library(tidyverse) #load tidyverse
+
+dad_mom <- read.delim("dad_mom.txt") #load data and assign as dataframe
+
+tidy_dad_mom <- dad_mom %>% #assign to new, tidy dataframe
+  unite(mom, name_mom, income_mom) %>% #mom data should be together
+  unite(dad, name_dad, income_dad) %>% #dad data should be together
+  gather(key = "parent", value = "data", mom, dad) %>% #sort things by parent
+  separate(col = "data", into = c("name", "income")) #separate name and income from each other
+```
+
+This has created a dataframe with four columns:
+
+  * fam_id = family number
+
+  * parent = dad or mom
+
+  * name = name of parent
+
+  * income = income of parent
+
+
+## QUESTION 2:
+
+THE GOAL: Join the five csv files together and answer questions about them
+
+```{r make CTRP dataframes}
+
+AUC_1 <- read.csv("CTRP_files/AUC_1.csv") #assign AUC_1.csv a dataframe
+AUC_2 <- read.csv("CTRP_files/AUC_2.csv") #assign AUC_2.csv a dataframe
+cancer_cell_line_info <- read.csv("CTRP_files/cancer_cell_line_info.csv") #assign cancer_cell_line_info.csv a dataframe
+compound_info <- read.csv("CTRP_files/compound_info.csv") #assign compound_info.csv a dataframe
+Experiment_info <- read.csv("CTRP_files/Experiment_info.csv") #assign Experiment_info.csv a dataframe
+```
+
+Observations about these files:
+
+  * AUC_1 and AUC_2 have all the same variables and could just be stitched together
+
+  * compound_info shares the master_cpd_id variable with AUC_1 and AUC_2
+
+  * Experiment_info shares the experiment_id variable with AUC_1 and AUC_2
+
+  * cancer_cell_line_info shares the master_ccl_id variable with Experiment_info
+
+  
+```{r combine CTRP dataframes into one}  
+#Join AUC_1 and AUC_2
+AUC <- AUC_1 %>% #assign as own dataframe for posterity/easy-undo
+  full_join(AUC_2, by = c("experiment_id", "area_under_curve", "master_cpd_id"))
+
+#Join AUC and compound_info
+AUC_compound <- AUC %>% #assign as own dataframe for posterity/easy-undo
+  full_join(compound_info, by = "master_cpd_id")
+
+#Join AUC_compound and Experiment_info
+AUC_compound_Experiment <- AUC_compound %>% #assign as own dataframe for posterity/easy-undo
+  full_join(Experiment_info, by = c("experiment_id" = "expt_id"))  
+
+#final join to put all together
+full_combo <- AUC_compound_Experiment %>% #assign as own dataframe for posterity/easy-undo
+  full_join(cancer_cell_line_info, by = "master_ccl_id")
+```
+
+## ANSWER THE FOLLOWING QUESTIONS (WITH GRAPHS!)
+
+**Which cancer type has the lowest AUC values to the compound "vorinostat"?**
+
+```{r solve vorinostat}
+#cpd_name will have to be "vorinostat"
+vorinostat_search <- full_combo %>% #assign to dataframe vorinostat_search
+  filter(cpd_name == "vorinostat") %>% #only look at vorinostat compound rows
+  arrange(area_under_curve) #order so that smallest AUC at top
```

Comment:
This is something I don't get from the `arrange()` verb because it only sorts the table, but when you print the graph, the order is completely lost.